src/balena-model.ts

```diff
@@ -736,11 +736,6 @@ export interface Device {
 			| [Release['Read']]
 			| []
 			| null;
-		should_be_running__release:
```

Comment:
???? I don't think that should have gone away

Reply:
It's a computed field, it cannot be written to

Reply:
Oh wait, it's also affected read typings 🤦